src/main/java/com/mineboundteam/minebound/magic/MagicEvents.java

```diff
@@ -84,6 +84,8 @@ public static void onButtonPress(InputEvent event) {
             MagicSync.NET_CHANNEL.sendToServer(new ButtonMsg(MsgType.PRIMARY_MENU));
         else if (ClientRegistry.SECONDARY_MAGIC_SELECT.consumeClick())
             MagicSync.NET_CHANNEL.sendToServer(new ButtonMsg(MsgType.SECONDARY_MENU));
+        else if(ClientRegistry.FIRE_UTILITY_SPELL_TOGGLE.consumeClick())
+            MagicSync.NET_CHANNEL.sendToServer(new ButtonMsg(MsgType.FIRE_UTILITY_TOGGLE));
```

Comment:
Don't `else if` this. Make it a separate if. Refer to my original comment about consolidating this check for why.